main.cpp

```diff
@@ -13,75 +13,486 @@ What you create in this project will be used as the basis of Project 5 in the co
+    int wattsPerHour = 200;
+    //Compress incoming audio
+    void compress (int ratio, float threshold);
+    //Turn on
+    void turnOn ();
+    //Turn off
+    void turnOff ();
+};
+    /*
+    Thing 8)
+    5 properties:Analog Reverb
+        1)Number of presets (int)
+        2)Number of largest size parameter (float)
+        3)Number of knobs (int)
+        4)Amount of money to buy (float)
+        5)colour (std::string)
+    3 things it can do:
+        1)Apply bright plate reverbs
+        2)Apply dark plate reverbs
+        3)Apply large room reverb
+    */
+struct analogReverb
+{
+    //Number of presets
+    int numOfpreSets = 200;
+    //)Number of largest size parameter
+    float numOfMaxSizeVerb = 1000;
+    //Number of knobs
+    int numOfKnobs = 12;
+        struct parameters
+        {
+            float roomSize = 100.f;
+            int decayInMiliSeconds = 10;
+            int presetNum = 1;
+            float delayInMiliSeconds = 100;
+            bool lowCutFilter = true;
+
+             void type (std::string type1 = "Plate", std::string type2 = "Spring");
+             void accessParameters ();
+             void resetParameters ();
+
+
+        };    
+    //Amount of money to buy 
+    double cost = 2999.99;
+    //colour
+    std::string colour = "Silver";
+    //Apply bright plate reverbs
+    void brightPlate ();
+    //Apply dark plate reverbs
+    void DarkPlate ();
+    //Apply large room reverb
+    void largeRoomVerb ();
+};
+    /*
+    Thing 9) Speakers
+    5 properties:
+        1)Number of Speakers (int)
+        2)Size of speakers in inches (int)
+        3)Number of rms watts per hour (int)
+        4)Number rof max watts per hour (int)
+        5)brand of speakers (std::string)
+    3 things it can do:
+        1)output incoming sound
+        2)Turn on
+        3)Control incoming sound to be louder
+    */
+ 
+struct speakers
+{
+    //Number of Speakers
+    int numOfSpeakers = 6;
+    //Size of speakers in inches
+    int size = 6;
+    //Number of rms watts per hour
+    int rmsPerHourWatts = 200;
+    //Number rof max watts per hour 
+    int maxWattsPerHour = 500;
+    //brand of speakers
+    std::string brand = "krk";
+    //output incoming sound
+    void soundOn ();
+    //Turn on
+    void turnOn ();
+    //Control incoming sound to be louder
+    int soundLouder ();
+};
+
+/*Thing 10) Music Studio
+5 properties:
+    1)Mixing Desk 
+    2)Imac
+    3)Analog Compressors
+    4)Analog Reverbs
+    5)Speakers
+3 things it can do:
+    1)Output sound
+    2)Mix music
+    3)Compress audio
 */
+struct musicStudio
+{
+//Mixing Desk
+mixingDesk Mixingdesk; 
+//Imac
+Imac imac;
+//Analog Compressors
+analogCompressor AnalogCompressor;
+//Analog Reverbs
+analogReverb AnalogReverb;
+//Speakers
+speakers Speakers;
+//Output sound
+bool outputSound (bool SoundOn = true, bool turnOn = true);
+//Mix music
+void mixMusic ();
+//Compress audio
+void compress (bool compress = true);
+};
```

Comment:
- fix this indentation
- variable names and function names need to start with a **lower-case** letter, **including function arguments**
- Type names need to start with an Upper case letter
- [ ] revise